.gitignore

```diff
@@ -105,6 +105,8 @@ ENV/
 
 # IntelaJ
 .idea/
+/snaps-boot.iml
```

Comment:
Are both of these lines needed?  Won't snaps-boot.iml get all instances of the file.

What about being more generic with *.iml?